src/controller.cpp

```diff
@@ -837,6 +838,13 @@ void Controller::statusUpdated(const QString& serverIpv4Gateway,
                                const QString& deviceIpv4Address,
                                uint64_t txBytes, uint64_t rxBytes) {
   logger.debug() << "Status updated";
+
```

Comment:
My understanding of the `Controller` and how it interracts with the rest of the VPN is that we only call the `getStatus()` method once after connection to fetch the gateway addresses and kick off the connection health monitoring. This means that we aren't really going to accumulate meaningful data as the counter will always be somewhere around zero.

What we need is something to fetch this data periodically, or to gather the data upon connection deactivation.

Reply:
Yeap, I thought I was being smart here, but I could not have been more wrong. You are right and I will change this.